Dockerfile.muti

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.18 AS builder
+
+WORKDIR /go/src/github.com/alibaba/open-local
+COPY . .
+RUN make build && chmod +x bin/open-local
+
+FROM openlocal/open-local-base:latest
+LABEL maintainers="Alibaba Cloud Authors"
+LABEL description="open-local is a local disk management system"
+COPY --from=builder /go/src/github.com/alibaba/open-local/bin/open-local /bin/open-local
```

Comment:
Looks restic tools is not included ?

Reply:
Hmmm... I miss it.  Fixed, Please re-check.